tests/release/pipelines/push_to_external_registry.go

```diff
@@ -172,10 +172,11 @@ var _ = framework.ReleasePipelinesSuiteDescribe("Push to external registry", Lab
 			component, err := fw.AsKubeAdmin.HasController.GetComponent(component.GetName(), devNamespace)
 			Expect(err).ShouldNot(HaveOccurred(), fmt.Sprintf("could not get component %s in the %s namespace", component.GetName(), devNamespace))
 			containerImageDigest := strings.Split(component.Spec.ContainerImage, "@")[1]
+			imageDigest := strings.Replace(containerImageDigest, ":", "-", -1)
```

Comment:
I'm a little confused by this. Isn't this checking that the image pushed here https://github.com/redhat-appstudio/release-service-catalog/blob/development/tasks/push-snapshot/push-snapshot.yaml#L136 exists? I don't see us doing this replace : with - in any of our tags that is why I am confused what tag is being checked here now

Reply:
The URL formats are different. Here I am checking the URL in format like quay.io/redhat-appstudio-qe/dcmetromap:sha256-xxx and in #L136, the format is like quay.io/redhat-appstudio-qe/dcmetromap@sha:xxx.

Reply:
When I checked the quay repo, I found sha-xxx tags are in the list. Anyway, I changed the URL to the format used in #L136.  Can you please to review it again?